src/einsteinpy/utils/bl_coord_transforms.py

```diff
@@ -0,0 +1,18 @@
+import astropy.units as u
```

Comment:
Why a new file?

Reply:
appending to the same file would trigger code-climate issues.
Plus, new file improves clarity and modularity

Reply:
code climate requires files having < ~250 lines of code